lib/r10k/git/rugged/working_repository.rb

```diff
@@ -117,11 +117,15 @@ def origin
     with_repo { |repo| repo.config['remote.origin.url'] }
   end
 
-  def dirty?
+  def dirty?(exclude_spec=false)
```

Comment:
I forget, is this default supposed to be `true` in 4.x?

Reply:
ooh hmm, not sure, I'll look

Reply:
looks like it is https://github.com/puppetlabs/r10k/blob/4.x/doc/dynamic-environments/configuration.mkd#exclude_spec 
good catch!

Reply:
updated!